### PR TITLE
Clarify UI fields like Ref and Revision

### LINKED ISSUE #62: "Finished" time is often in the future
**Describe the bug**
Screenshot 2020-02-21 at 08 24 14

**To Reproduce**
1. Start a build and let it finish
2. Sometimes(?) the "Finished" time is in the future

**Expected behavior**
I think the "Finished" time should be in the past if the build is finished.

Or, if it's a prediction, I think it should be called "Estimated to finish".

### PR DESCRIPTION
**Pull request by jankeromnes**
By:
- ~~renaming `Finished` to `Duration`~~
- adding helpful tooltips to `Ref` and `Revision` in the UI

### CODE REVIEW

File: cmd/client/job-get.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ Metadata:
   Owner:	{{ .Metadata.Owner }}
   Trigger:	{{ .Metadata.Trigger }}
   Started:	{{ .Metadata.Created | toRFC3339 }}
-  Finished:	{{ .Metadata.Finished | toRFC3339 }}
+  Duration:	{{ .Metadata.Duration | toRFC3339 }}
```
Comment (jankeromnes) [inline cmd/client/job-get.go]: Oops, it looks like sometimes, `Finished` actually is the finished time and not a duration:

```
Metadata:
  Owner:           Jan Keromnes
  Trigger:         TRIGGER_MANUAL
  Started:         2020-02-21T10:19:58Z
  Finished:        2020-02-21T10:21:08Z
```

Comment (csweichel) [reply cmd/client/job-get.go]: Finished always is the finished time :)
It's just the UI that shows it as duration.

**Comment by jankeromnes** [comment]
Removed contentious bits & rebased. Should be good to merge now. 😇

@csweichel Please take a look when you have time.

**Review by csweichel — APPROVED** [review]
Thanks. Lgtm

**Comment by felladrin** [comment]
Is the change about renaming "Finished" to "Duration" still in place?

In this Werft job details page for example, I see "Finished" on the top bar: https://werft.gitpod-dev.com/job/gitpod-build-main.3195

image

**Pull request merged**